.eslintrc.json

```diff
@@ -0,0 +1,121 @@
+{
+    "root": true,
+    "ignorePatterns": ["projects/**/*"],
```

Comment:
why ignore the projects?

Reply:
I think it ignores the projects in the root and then has one file for each project that extends this one.

It was generated automatically, so I don't have full insight, but it checks every project in the end.